javascript/node/selenium-webdriver/lib/test/fileserver.js

```diff
@@ -106,6 +106,7 @@ const Pages = (function () {
   addPage('webComponents', 'webComponents.html')
   addPage('xhtmlTestPage', 'xhtmlTest.html')
   addPage('uploadInvisibleTestPage', 'upload_invisible.html')
+  addPage('virtualAuthenticator', 'virtualAuthenticator.html')
```

Comment:
Please update to match the same file as the one created in the `py` PR.

Reply:
I have made the changes. But I can't make a PR before [https://github.com/SeleniumHQ/selenium/pull/10653] gets merged. The changes made in `virtual-authenticator.html` are required to successfully run the test.

Reply:
@AutomatedTester required changes have been made.